cpr/session.cpp

```diff
@@ -291,6 +292,14 @@ void Session::Impl::SetLowSpeed(const LowSpeed& low_speed) {
     }
 }
 
+void Session::Impl::SetLimitRate(const LimitRate& limit_rate) {
+    auto curl = curl_->handle;
+    if (curl) {
+        curl_easy_setopt(curl, CURLOPT_MAX_RECV_SPEED_LARGE, limit_rate.rate);
```

Comment:
Can you change LimitRate to have separate send and recv settings.

Reply:
sorry for the very very late reply, I've just added the changes you asked for here.